changelog/_unreleased/2021-05-27-make-use-of-abstraction-of-the-contextswitchroute.md

```diff
@@ -0,0 +1,10 @@
+---
+title: Make use of abstraction of the ContextSwitchRoute
+issue: noticket
+author: Steffen Beisenherz
+author_email: s.beisenherz@kellerkinder.de 
+author_github: Sironheart
+___
+# Storefront
+*  The AccountOrderController now does not require the shopware implementation of the ContextSwitchRoute anymore, 
```

Comment:
Hey @Sironheart, could you reword this? i often do that myself when everything else is ok, but as written in the [changelog info](https://github.com/shopware/platform/blob/trunk/adr/2020-08-03-Implement-New-Changelog.md) we do have 4 keywords to be used (Added, Removed, Changed, Deprecated). 
"Changed the AccountOrderController so it now does not require the shopware implementation of the ContextSwitchRoute anymore, instead accepting any implementation of the AbstractContextSwitchRoute" ?

Reply:
done so